test/DebugInfo/resilience.swift

```diff
@@ -9,31 +9,38 @@
 // RUN: %target-swift-frontend -g -I %t -emit-ir -enable-resilience %s  -o - \
 // RUN:  | %FileCheck %s
 //
-// RUN: %target-swift-frontend -g -I %t -emit-sil -enable-resilience %s -o - \
-// RUN:    | %FileCheck %s --check-prefix=CHECK-SIL
-// RUN: %target-swift-frontend -g -I %t -emit-sil -enable-resilience %s -o - \
+// RUN: %target-swift-frontend -g -I %t -emit-ir -enable-resilience %s -o - \
 // RUN:    -debugger-support | %FileCheck %s --check-prefix=CHECK-LLDB
 import resilient_struct
 
-func use<T>(_ t: T) {}
-
+// CHECK-LABEL: define swiftcc void @"$S10resilience1fyyF"()
+// CHECK-LLDB-LABEL: define swiftcc void @"$S10resilience1fyyF"()
 public func f() {
   let s1 = Size(w: 1, h: 2)
-  use(s1)
+  takesSize(s1)
   // CHECK: %[[ADDR:.*]] = alloca i8*
   // CHECK: call void @llvm.dbg.declare(metadata i8** %[[ADDR]],
   // CHECK-SAME:                        metadata ![[V1:[0-9]+]],
   // CHECK-SAME:                        metadata !DIExpression(DW_OP_deref))
   // CHECK: %[[S1:.*]] = alloca i8,
   // CHECK: store i8* %[[S1]], i8** %[[ADDR]]
-  // CHECK: ![[V1]] = !DILocalVariable(name: "s1", {{.*}}type: ![[TY:[0-9]+]])
-  // CHECK: ![[TY]] = !DICompositeType(tag: DW_TAG_structure_type, name: "Size",
-  // FIXME-NOT: size:
-  // CHECK: =
+
+  // CHECK-LLDB: %[[ADDR:.*]] = alloca %T16resilient_struct4SizeV
+  // CHECK-LLDB: call void @llvm.dbg.declare(metadata %T16resilient_struct4SizeV* %[[ADDR]],
+  // CHECK-LLDB-SAME:                        metadata ![[V1:[0-9]+]],
+  // CHECK-LLDB-SAME:                        metadata !DIExpression())
 }
 
-// CHECK-SIL: // f()
-// CHECK-LLDB: // f()
-// CHECK-SIL: %0 = alloc_stack $Size, let, name "s1"
-// CHECK-LLDB: %0 = metatype $@thin Size.Type
-// CHECK-LLDB: debug_value %{{.*}} : $Size, let, name "s1"
+
+// CHECK-LABEL: define swiftcc void @"$S10resilience9takesSizeyy16resilient_struct0C0VF"(%swift.opaque* noalias nocapture)
+// CHECK-LLDB-LABEL: define swiftcc void @"$S10resilience9takesSizeyy16resilient_struct0C0VF"(%T16resilient_struct4SizeV* noalias nocapture dereferenceable(16))
+public func takesSize(_ s: Size) {}
+
+
+// CHECK: ![[V1]] = !DILocalVariable(name: "s1", {{.*}}type: ![[TY:[0-9]+]])
+// CHECK: ![[TY]] = !DICompositeType(tag: DW_TAG_structure_type, name: "Size",
+// CHECK: =
+
+// CHECK-LLDB: ![[V1]] = !DILocalVariable(name: "s1", {{.*}}type: ![[TY:[0-9]+]])
+// CHECK-LLDB: ![[TY]] = !DICompositeType(tag: DW_TAG_structure_type, name: "Size",
+// CHECK-LLDB: =
```

Comment:
What's up with this check line, and the one above for non-LLDB?

Reply:
I don't know, I'll remove it unless @adrian-prantl objects.